src/libsync/propagateremotedeleteencryptedrootfolder.h

```diff
@@ -27,7 +27,7 @@ class PropagateRemoteDeleteEncryptedRootFolder : public AbstractPropagateRemoteD
 public:
     PropagateRemoteDeleteEncryptedRootFolder(OwncloudPropagator *propagator, SyncFileItemPtr item, QObject *parent);
 
-    virtual void start() Q_DECL_OVERRIDE;
+    void start() Q_DECL_OVERRIDE;
```

Comment:
@mgallien Don't we want to change `Q_DECL_OVERRIDE` to just `override`, for consistency?